.mill-version

```diff
@@ -0,0 +1 @@
+0.9.4-29-9d1d6f
```

Comment:
Can you update the version in the `mill` script instead ? Just for consistency with the other repos.

Reply:
Sure, but then we rely on people always using the local mill wrapper, and it may lead to inconsistencies if people use a system-installed mill wrapper. I don't really mind either way.

Reply:
The normal `mill` executable actually scrapes the local `./mill` file for the version, so it should behave the same way either way